wrappers/android/librealsense/src/main/java/com/intel/realsense/librealsense/Option.java

```diff
@@ -91,6 +91,7 @@
     AUTO_GAIN_LIMIT(86),
     AUTO_RX_SENSITIVITY(87),
     OPTION_TRANSMITTER_FREQUENCY(88);
+    RS2_OPTION_VERTICAL_BINNING(88);
```

Comment:
No good -- you have a semicolon in the previous line.
Please run a Jenkins build with your code as we have no Travis to check Linux now.

Reply:
done